packages/react-sdk/src/css/presets.ts

```diff
@@ -25,3 +25,102 @@ export const outline: Styles = [
     value: { type: "unit", value: 1, unit: "px" },
   },
 ];
+
+export const margins = [
+  {
+    property: "marginTop",
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+  {
+    property: "marginRight",
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+  {
+    property: "marginBottom",
+    value: { type: "unit", value: 0, unit: "px" },
+  },
```

Comment:
why number and px?

Reply:
typo, please change to number, I am off right now